### ISSUE TITLE
The meta-data is invalid or is corrupt problem.

### ISSUE THREAD

**Issue by lieshanhe** [nguyenducduy/phalcon-jumpstart#6]
Hi , May I know how did you work it out ?

https://forum.phalconphp.com/discussion/7335/phalcon-203-the-metadata-is-invalid-or-is-corrupt

**Comment by nguyenducduy** [comment]
it problem occured when running out of phalcon version updated, i was update source for lastest version.

**Comment by lieshanhe** [comment]
Yes, I used the latest version of Phalcon, but this problem is still occured.

**Comment by nguyenducduy** [comment]
@@ it occured because i did not defined default value of meta data annotations, add this code to fix it.
<code>
/_Files AnnotationsMetaDataInitializer.php_/

public function getMetaData(ModelInterface $model, DiInterface $di)
{
...
            // Default values for columns
            PhalconMetadata::MODELS_DEFAULT_VALUES => [],

```
        // Fields that allow empty strings
        PhalconMetadata::MODELS_EMPTY_STRING_VALUES => []
```

}
</code>
